pkg/ccl/testccl/sqlccl/show_transfer_state_test.go

```diff
@@ -20,6 +20,7 @@ import (
 	"github.com/cockroachdb/cockroach/pkg/testutils/serverutils"
 	"github.com/cockroachdb/cockroach/pkg/testutils/sqlutils"
 	"github.com/cockroachdb/cockroach/pkg/util/leaktest"
+	"github.com/jackc/pgx/v4"
```

Comment:
Can you use v5 or is v4 required for this test?

Reply:
done